app/strings.php

```diff
@@ -186,12 +186,12 @@
             __('Preview'),
             __('Publish project'),
             __('Once you publish your project, others can find it.'),
-            __(':language translation of project goals'),
+            __('Add :language translation of project goals'),
```

Comment:
I wonder if all/many of these can be removed. Do you know if the parts after "Add :language translation of" should have been token for interpolation like "Add :language translations of :label" and is already handled in the translatable-input.blade.php and translatable-textarea.blade.php files.

Reply:
Right, I think parts after "Add :language translation of" are handled by translatable components; the components use short label or label property for the after part, and current use cases all provide either label or short label.

Reply:
@chosww thanks for confirming that. I think we should remove these strings from the string placeholder file then, if they aren't necessary and are already handled.